src/app/ApplicationTemplate.UITests/Constants.cs

```diff
@@ -4,7 +4,7 @@ namespace ApplicationTemplate.UITests;
 
 public static class Constants
 {
-	public const string IOSAppName = "com.nventive.internal.applicationtemplate";
```

Comment:
Why remove `internal`???
Now iOS and Android don't match anymore.

Reply:
com.nventive.internal.applicationtemplate is in apple enterprise account, we can't use it in the individual account, for android and ios to match we'll have to change both of them.